projects/goat-protocol/index.js

```diff
@@ -16,6 +19,7 @@ module.exports = {
         if(pools[i].platformId != 'goatfi')
           vaults.push(pools[i].earnedTokenAddress);
       return yieldHelper({ vaults, chain, block, tokenAPI, useDefaultCoreAssets: true, })
-    }
+    },
+    staking: staking(REWARD_POOL_CONTRACT, GOA_TOKEN_CONTRACT, "arbitrum")
```

Comment:
not sure what bentura meant, but how can we price the token? is it already listed on coingecko?

Reply:
Yes, listed on coingecko: https://www.coingecko.com/en/coins/goat-protocol